src/__tests__/Panel.spec.js

```diff
@@ -12,4 +12,10 @@ describe('<Panel />', () => {
     const div = document.createElement('div');
     ReactDOM.render(<Panel />, div);
   });
+  it('matches snapshot', () => {
```

Comment:
Include a `describe` in here just like you did with the other tests to keep it consistent.